super-mode-calculator/src/lib/dynamoV2.ts

```diff
@@ -0,0 +1,126 @@
+import type { AWSError } from 'aws-sdk';
```

Comment:
what's the difference between this file and the original dynamo.ts?

Reply:
This is for the new table `super-mode-calculator-${stage.toUpperCase()}` ,I wanted to keep it separate from the code for existing table

Reply:
👍 ok, so eventually the old file will be deleted and this can be renamed